### PR TITLE
Learn HTML (chapter 6-11)

### PR DESCRIPTION
**Pull request by AaronForinton**
Fixes: #7694 
Fixes: #7693 
Fixes: #7696 
Fixes: #7697 
Fixes: #7698 
Fixes: #7699 

Learn HTML

Ch 6 - Attributes
Ch 7 - Text basics
Ch 8 - Links
Ch 9 - Lists
Ch 10 - Navigation
Ch 11 - Tables

### CODE REVIEW

File: src/site/content/en/learn/html/lists/index.md [hunk 1]
```diff
@@ -3,10 +3,290 @@ title: 'Lists'
...
+item elements. Let's create a list of machines. We use an unordered list because the order doesn't matter (don't tell them that):
+
+```html
+<ul>
+   <li>Blender</li>
+   <li>Toaster</li>
+   <li>Vacuum</li>
+</ul>
+```
+<img src="image/cGQxYFGJrUUaUZyWhyt9yo5gHhs1/OeZKatnZZPbs7q0yuM2C.png" width="196" height="146" alt="Blender, toaster, vacuum">
```
Comment (AaronForinton) [inline src/site/content/en/learn/html/lists/index.md:42]: @rachelandrew Images are my major issue at this point - especially for this one, [where they're supposed to be alongside the HTML](https://docs.google.com/document/d/14c4qiKoR7aRfec7Sup66Ag5jbCS599HXSSUQMSkwBXc/edit#heading=h.3dy7ebarqnns). Personally I can't get images to appear at anything other than full size, and when I tried to implement it with HTML nothing comes up at all.

File: src/site/content/en/learn/html/text-basics/index.md [hunk 2]
```diff
@@ -3,10 +3,198 @@ title: 'Text basics'
...
+and `<h6>` the least. For many years, developers were told that headings were used by browsers to outline documents.
+That was originally a goal, but browsers haven't implemented outlining features. However, screen reader users do use headings
+as an exploration strategy to learn about the content of the page, navigating through headings with the `h` key. So ensuring
+that heading levels are implemented as you would outline a document makes your content accessible and is still very much encouraged.
+
+By default, browsers style `<h1>` the largest, `<h2>` slightly smaller, with each subsequent heading level being smaller
+by default. Interestingly, browsers by default also decrement the `<h1>` font size based on how many `<article>`, `<aside>`, `<nav>`, or
+`<section>` elements it is nested in.
+
+{% Img src="image/cGQxYFGJrUUaUZyWhyt9yo5gHhs1/ktRJF3DKyFHtfLifJXkn.png", alt="Nested H1 examples", width="242", height="548" %}
```
Comment (AaronForinton) [inline src/site/content/en/learn/html/text-basics/index.md:30]: This image is absolutely enormous and low-res despite me trying to adjust its size parameters

File: src/site/content/en/learn/html/text-basics/index.md [hunk 3]
```diff
@@ -3,10 +3,198 @@ title: 'Text basics'
...
+If you take a look at ToastyMcToastface's workshop review (not included in the code sample above), there are some unusual text characters:
+
+```html
+<blockquote>Learning with Hal and Eve exceeded all of my wildest fantasies. All they did was stick a USB in. They promised that it was a brand new USB, so we know there were no viruses on it. The Russians had nothing to do with it. This has no̶̼͖ţ̘h̝̰̩͈̗i̙̪n͏̩̙͍̱̫̜̟g̢̣ͅ ̗̰͓̲̞̀t͙̀o̟̖͖̹̕ ͓̼͎̝͖̭dó̪̠͕̜ ͍̱͎͚̯̟́w̮̲̹͕͈̟͞ìth̢ ̰̳̯̮͇</blockquote>
+```
+
+The last sentence in this blockquote can also be written as:
+
+```html
+This has no&#x336;&#x33C;&#x356;&tcedil;&#x318;h&#x31D;&#x330;&#x329;&#x348;&#x317;i&#x319;&#x32A;n&#x34F;&#x329;&#x319;&#x34D;&#x331;&#x32B;&#x31C;&#x31F;g&#x322;&#x323;&#x345; &#x317;&#x330;&#x353;&#x332;&#x31E;&#x300;t&#x359;&#x300;o&#x31F;&#x316;&#x356;&#x339;&#x315; &#x353;&#x33C;&#x34E;&#x31D;&#x356;&#x32D;d&oacute;&#x32A;&#x320;&#x355;&#x31C; &#x34D;&#x331;&#x34E;&#x35A;&#x32F;&#x31F;&#x301;w&#x32E;&#x332;&#x339;&#x355;&#x348;&#x31F;&#x35E;&igrave;th&#x322; &#x330;&#x333;&#x32F;&#x32E;&#x347;
```
Comment (AaronForinton) [inline src/site/content/en/learn/html/text-basics/index.md:186]: Would it be preferable for this to clip onto multiple lines?

File: src/site/content/en/learn/html/lists/index.md [hunk 4]
```diff
@@ -3,10 +3,290 @@ title: 'Lists'
...
+   <li>Blender</li>
+   <li>Toaster</li>
+   <li>Vacuum</li>
+</ol> https://codepen.io/estelle/pen/OJEXLWK
+
+<ol type="A">
+   <li>Blender</li>
+   <li>Toaster</li>
+   <li>Vacuum</li>
+</ol> https://codepen.io/estelle/pen/OJEXLWK
```
Comment (AaronForinton) [inline src/site/content/en/learn/html/lists/index.md:64]: Are these codepen links meant to be here?

File: src/site/content/en/learn/html/text-basics/index.md (hunk 2 above)
Comment (AaronForinton) [reply src/site/content/en/learn/html/text-basics/index.md:30]: https://deploy-preview-9143--web-dev-staging.netlify.app/learn/html/text-basics/

File: src/site/content/en/learn/html/lists/index.md (hunk 4 above)
Comment (estelle) [reply src/site/content/en/learn/html/lists/index.md:64]: i added the codepen links to show the resource if y'all wanted to include a codepen; they should not be displayed as text though.

File: src/site/content/en/learn/html/lists/index.md (hunk 1 above)
Comment (estelle) [reply src/site/content/en/learn/html/lists/index.md:42]: if it doesn't work, putting a codepen instead is an option.

File: src/site/content/en/learn/html/text-basics/index.md (hunk 3 above)
Comment (estelle) [reply src/site/content/en/learn/html/text-basics/index.md:186]: yes

File: src/site/content/en/learn/html/lists/index.md (hunk 1 above)
Comment (AaronForinton) [reply src/site/content/en/learn/html/lists/index.md:42]: @estelle I think it's easy to make it work but I was just wondering if @rachelandrew knew the best way to put them side by side

File: src/site/content/en/learn/html/lists/index.md (hunk 4 above)
Comment (AaronForinton) [reply src/site/content/en/learn/html/lists/index.md:64]: @rachelandrew Let me know what you think about including these codepens! 

Thanks for the comments @estelle I'll make sure to get this stuff updated on Monday.

**Review by rachelandrew — CHANGES_REQUESTED** [review]
Bunch of changes, there are also some big blocks of code I think would be better as CodePens, for example the step by step building up the tables. If I didn't give you the log in details for the CodePen account let me know and I can share them with you.

Hoping to publish this on Thursday as Learn Accessibility new modules are landing today.

File: src/site/content/en/learn/html/lists/index.md [hunk 5]
```diff
@@ -3,10 +3,298 @@ title: 'Lists'
...
+</li>
+</ul>
+```
+
+Nesting lists within lists is also very common. While MLW doesn't have any nested lists, this site does. In the first chapter of this series,
+Overview of HTML, the main elements section has two subsections. In the table of contents, which is an unordered list, there is a nested
+unordered list with links to these two sections:
+
+```html
+<ul role="list">
```
[the remark this answers is not in the document]
Comment (AaronForinton) [reply src/site/content/en/learn/html/lists/index.md:220]: Sorry, this looks normal in my IDE... looking at it!

File: src/site/content/en/learn/html/lists/index.md (hunk 1 above)
Comment (AaronForinton) [reply src/site/content/en/learn/html/lists/index.md]: All these images are broken and I'm not sure why. Also I've implemented columns - do you know how to get the HTML to show up un-rendered inside them?

File: src/site/content/en/learn/html/tables/index.md [hunk 6]
```diff
@@ -3,10 +3,329 @@ title: 'Tables'
...
+
+The caption appears outside the table. The location of the caption can be set with the CSS [`caption-side`](https://developer.mozilla.org/docs/Web/CSS/caption-side) property, which is a
+better practice than using the deprecated `align` attribute. This can set the caption to the top and bottom. The left and right
+side positioning, with `inline-start` and `inline-end`, are not yet fully supported. Top is the default browser presentation.
+
+```html
+<table>
+  <caption>MLW Students</caption>
+</table>
+```
```
Comment (rachelandrew) [inline src/site/content/en/learn/html/tables/index.md:61]:
```suggestion
{% Codepen {
  user: 'web-dot-dev',
  id: 'yLEZRyW',
  height: 300,
  theme: 'dark',
  tab: 'html,result'
} %}
```

File: src/site/content/en/learn/html/lists/index.md [hunk 7]
```diff
@@ -3,10 +3,330 @@ title: 'Lists'
...
+<div class="switcher">
+  <figure>
+      <ul>
+      <li>Blender</li>
+      <li>Toaster</li>
+      <li>Vacuum</li>
+    </ul>
+  </figure>
+  <figure>
+    <img src="image/cGQxYFGJrUUaUZyWhyt9yo5gHhs1/fLffpFWfFQHTc3pbNBbk.png", alt="Blender, toaster, vacuum, listed with bullet points.", width="212", height="142">
```
Comment (rachelandrew) [inline src/site/content/en/learn/html/lists/index.md:43]:
```suggestion
    <img src="https://web-dev.imgix.net/image/cGQxYFGJrUUaUZyWhyt9yo5gHhs1/fLffpFWfFQHTc3pbNBbk.png", alt="Blender, toaster, vacuum, listed with bullet points.", width="212", height="142">
```

**Review by rachelandrew — APPROVED** [review]

**Pull request merged**